### ISSUE TITLE
Get Context works erratically with XYplorer

### ISSUE THREAD

**Issue by erohtar** [adirh3/Fluent-Search#488]
**Describe the bug**
Sometimes FS gets current path correctly.
Sometimes FS gets current title bar that includes application name and version.
Sometimes FS gets the path to a tab I was on earlier, but not the current one.

This was correct:
![image](https://user-images.githubusercontent.com/5120628/152595928-c84902b7-c21e-4da9-a829-5fcab755c490.png)

This was both wrong path and included extra text from titlebar
![image](https://user-images.githubusercontent.com/5120628/152596126-3c4b40aa-8b61-46ec-9dad-e0fde5bde701.png)

**To Reproduce**
Import this Task and press Ctrl+Alt+Q while switching folders/tabs in XYplorer.

```
zoom: 0.7708172725923621
offsetX: 516.37659767580055
offsetY: -309.73515111041922
children:
- id: 0
  type: Hotkey
  model:
    hotkey:
      name: Hotkey Task trigger
      hotKeys:
      - LCTRL
      - LSHIFT
      - F
  position:
    x: 480
    y: 1648
  variables:
  - name: myvar
    value: result
  children:
  - id: 1
    type: Get context
    model: {}
    position:
      x: 684
      y: 1648
    variables:
    - name: folder
      value: result
    children:
    - id: 2
      type: Show and search
      model:
        searchText: ''
        searchTag: '{Regex.Replace(folder, " - XYplorer .*", "")}'
        searchType: SearchAll
      position:
        x: 894
        y: 1645
      variables:
      - name: myvar
        value: result
      children: []
name: Current Folder Search
author: erohtar
iconGlyph: 
description: None
enable: true

```

**Comment by adirh3** [comment]
This issue has been potentially fixed in version 0.9.91.27                                         \nChangelog - https://install.appcenter.ms/users/adirh3-gmail.com/apps/fluent-search-alpha/distribution_groups/exe

**Comment by erohtar** [comment]
Seems to be working well now. Thanks @adirh3, this one was really important to me.
